khl/client.py

```diff
@@ -363,3 +369,29 @@ async def fetch_blocked_friends(self) -> List[Friend]:
 
     async def start(self):
         await asyncio.gather(self.handle_pkg(), self.gate.run(self._pkg_queue))
+
+    async def message_view(self, msg_id: str = '') -> Dict:
+        """get message detail by message id"""
+        if len(msg_id) > 0:
+            return await self.gate.exec_req(api.Message.view(msg_id))
+
+    async def list_messages(self,
+                            target_id: str = None,
```

Comment:
2. target id is required param in [khl api doc](https://developer.kookapp.cn/doc/http/message#%E8%8E%B7%E5%8F%96%E9%A2%91%E9%81%93%E8%81%8A%E5%A4%A9%E6%B6%88%E6%81%AF%E5%88%97%E8%A1%A8:~:text=%E8%AF%B4%E6%98%8E-,target_id,-string)

thus no default value for this